src/components/SignUp/SignUp.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../style/constants';
+
+label {
+  margin-top: 1rem !important;
+}
+
+.signup {
+  text-transform: uppercase;
+  margin-top: 6.75rem;
+}
+
+.confirmation__first_title {
+  font-family: Open Sans Bold;
+  font-weight: bold;
```

Comment:
In general it's better to specify which weight corresponds to "bold" it could be 400 or 600 depending on the needs and selected fonts